src/e2e/java/teammates/e2e/cases/e2e/InstructorCourseDetailsPageE2ETest.java

```diff
@@ -0,0 +1,124 @@
+package teammates.e2e.cases.e2e;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.CourseAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.AppUrl;
+import teammates.common.util.Const;
+import teammates.e2e.pageobjects.InstructorCourseDetailsPage;
+import teammates.e2e.util.TestProperties;
+
+/**
+ * SUT: {@link Const.WebPageURIs#INSTRUCTOR_COURSE_DETAILS_PAGE}.
+ */
+public class InstructorCourseDetailsPageE2ETest extends BaseE2ETestCase {
+    private StudentAttributes student;
+    private CourseAttributes course;
+
+    private String fileName;
+
+    @Override
+    protected void prepareTestData() {
+        testData = loadDataBundle("/InstructorCourseDetailsPageE2ETest.json");
+        student = testData.students.get("charlie.tmms@ICDetailsE2eT.CS2104");
+        if (!TestProperties.isDevServer()) {
+            student.email = TestProperties.TEST_STUDENT1_ACCOUNT;
+        }
+
+        removeAndRestoreDataBundle(testData);
+        course = testData.courses.get("ICDetailsE2eT.CS2104");
+        fileName = "/" + course.getId() + "_studentList.csv";
+    }
+
+    @BeforeClass
+    public void classSetup() {
+        deleteDownloadsFile(fileName);
+    }
+
+    @Test
+    public void allTests() {
+        AppUrl detailsPageUrl = createUrl(Const.WebPageURIs.INSTRUCTOR_COURSE_DETAILS_PAGE)
+                .withUserId(testData.instructors.get("ICDetailsE2eT.instr").googleId)
+                .withCourseId(course.getId());
+        InstructorCourseDetailsPage detailsPage = loginAdminToPage(detailsPageUrl, InstructorCourseDetailsPage.class);
+
+        ______TS("verify loaded details");
+        InstructorAttributes[] instructors = { testData.instructors.get("ICDetailsE2eT.instr"),
+                testData.instructors.get("ICDetailsE2eT.instr2"),
+        };
+        StudentAttributes[] students = { testData.students.get("alice.tmms@ICDetailsE2eT.CS2104"),
```

Comment:
Nitpicking here. Looks kind of weird that one item is beside the `{`, can put everything in new line instead? Same goes for the `instructors` variable above.